eng/pipelines/docindex.yml

```
@@ -40,7 +40,15 @@ jobs:
           filePath: eng/common/scripts/Update-DocsMsPackages.ps1
           arguments: -DocRepoLocation $(DocRepoLocation)
         displayName: Update Docs Onboarding
-
+        condition: >-
+          and(
+            succeeded(),
+            or(
+              eq(variables['Build.Reason'], 'Schedule'),
+              eq(variables['Force.MainUpdate'], 'true')
+            )
+          )
+          
```

```suggestion
        condition: and(succeeded(), or(eq(variables['Build.Reason'], 'Schedule'), eq(variables['Force.MainUpdate'], 'true')))
```

While what you had is correct we generally have been doing these on one line.